bundles/org.openhab.binding.nuki/README.md

```diff
@@ -34,35 +34,146 @@ The following configuration options are available:
 | apiToken        | The API Token which you configured during [Initial Bridge setup](https://nuki.io/en/support/bridge/bridge-setup/initial-bridge-setup/).                                                            | Required     |
 | manageCallbacks | Let the Nuki Binding manage the callbacks on the Nuki Bridge. It will add the required callback on the Nuki Bridge. If there are already 3 callbacks, it **will delete** the callback with ID `0`. | Default true |
+| secureToken     | Whether hashed token should be used when communicating with Nuki Bridge. If disabled, API token will be sent in plaintext with each request.                                                       | Default true |
+
+### Bridge discovery
+
+Bridges on local network can be discovered automatically if both Nuki Bridge and openHAB have working internet connection. You can check whether discovery
+is working by checking [discovery API endpoint](https://api.nuki.io/discover/bridges). To discover bridges do the following:
+
+* In openHAB UI add new thing, select Nuki Binding and start scan. LED on bridge should light up.
+* Within 30s press button on Nuki Bridge you want to discover.
+* Bridge should appear in inbox.
+
+Pressing bridge button is required for binding to obtain valid API token. If the button isn't pressed during discovery, bridge will
+be created but token must be set manually for binding to work.
+
 
 ## Supported Things
 
-This binding support just one thing type: The Nuki Smart Lock. Create one `smartlock` per Nuki Smart Lock available in you home automation environment.
+This binding supports 2 things - Nuki Smart Lock and Nuki Opener. Both devices can be added using discovery after bridge they are 
+connected to is configured and online.
+
+### Nuki Smart Lock
 
 The following configuration options are available:
 
 | Parameter | Description                                                                                                                                                                                               | Comment       |
 | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| nukiId    | The `Nuki-ID` of the Nuki Smart Lock. It is a 8-digit hexadecimal string. Look it up on the sticker on the back of the Nuki Smart Lock (remove mounting plate).                                           | Required      |
 | unlatch   | If set to `true` the Nuki Smart Lock will unlock the door but then also automatically pull the latch of the door lock. Usually, if the door hinges are correctly adjusted, the door will then swing open. | Default false |
 
-## Supported Channels
+#### Supported Channels
```

Comment:
Would be awesome if you could put these into a table like in the other bindings.

Reply:
But I'd like to keep the lists of supported actions and states for some channels and I don't know how to fit them into single table. I looked at other binding and for example [Astro has this big multilevel list](https://www.openhab.org/addons/bindings/astro/#channels), other bindings use table but they don't need to specify supported actions.

Reply:
You could put the *command options* in a dedicated column and shorten them to e.g. "2=Unlock, 7=Unlatch".

Please note that Actions are something different. See https://www.openhab.org/docs/configuration/actions.html

Reply:
I've reformatted channel list as a table and put available commands into separate tables